gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/BigQuery.java

```diff
@@ -138,25 +126,19 @@ static String selector(TableField... fields) {
     STATUS("status"),
     USER_EMAIL("user_email");
 
+    static final List<FieldSelector> REQUIRED_FIELDS =
```

Comment:
Fine as is. Just want to mention an alternative:

```java
static final List<? extends FieldSelector> REQUIRED_FIELDS =
        ImmutableList.of(JOB_REFERENCE, CONFIGURATION);
```

and changing all the FieldSelector.SelectorHelper method signature to accept `List<? extends FieldSelector>`).